spec/lib/vcr/library_hooks/net_http2_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+require 'vcr/library_hooks/net_http2'
+
+RSpec.describe "Net::HTTP/2 hook" do
+  it 'records request' do
+    recorded, played_back = [1, 2].map do
+      VCR.use_cassette('nghttp2', :record => :once) do
+        client = NetHttp2::Client.new("http://nghttp2.org")
+        response = client.call(:get, '/')
```

Comment:
```suggestion
        response = client.call(:get, '/')
        client.close
```

I think you're missing closing the client connection. I was able to see the errors locally when running this branch locally.

Reply:
Nice catch @andrehjr! Thanks a lot!